wiki/Home.md

```diff
@@ -0,0 +1,26 @@
+
+Welcome to the unified-codes wiki!
+
+"unified-codes" is intended to complately replace the original universal drug codes server. 
+* The "original" universal drug codes server is currently located at https://sussol.github.io/universal-drug-codes-server/.
+* "unified-codes" (aka V2) is not yet deployed.
+
+## System Overview/Design [TO COMPLETE]
+<!-- This is quite high level - A diagram may be useful -->
+* Database 
```

Comment:
I think might be better to keep the database and GraphQL layer together (since they function as one service, i.e. the data service)? I think easier to understand and more consistent with the codebase and other documentation.